autoload/autoload.php

```diff
@@ -15,14 +15,13 @@
 use function class_alias;
 use function str_replace;
 use function strstr;
+use function version_compare;
 
 /** @psalm-suppress DeprecatedClass */
 $version = strstr(Versions::getVersion('symfony/console'), '@', true) ?: '';
 
-if (str_replace('v', '', $version) >= '5.0.0') {
-    // phpcs:ignore WebimpressCodingStandard.PHP.CorrectClassNameCase
+if (version_compare(str_replace('v', '', $version), '5.0.0', '>=')) {
     class_alias(ContainerCommandLoaderTypeHint::class, ContainerCommandLoader::class);
 } else {
-    // phpcs:ignore WebimpressCodingStandard.PHP.CorrectClassNameCase
```

Comment:
I'd prefer to keep it here instead of excluding that rule globally. I am aware of the limitation when we using class_alias and class does not exist - then PHP thinks the class name is lowercase and so that sniff suggest it lowercased. I haven't found a way to detect that case and skip in the sniff.

Reply:
I personally would avoid having multiple ignore lines just because we think that this rule is somehow necessary in this project.
I can revert this and add dozens of `phpcs:ignore` but I am pretty sure that this will become a problem again somewhen in another PR.

Reply:
The rule is valuable for long-term autoloading error prevention, whereas `class_alias()` stuff has always been the edge case.

Reply:
Yes, but as we do reviews anyways, having that rule ignored in projects wont be a problem unless we fail reviewing features which implement new classes with invalid namespaces. There are other rules which gives more value than this one. 

But anyways, I'll revert these changes as I don't want to waste my time with discussions like this.
Expect a PR later this day.